src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,36 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public static int totalTasks = 0;
```

Comment:
The name of the int variable is a bit misleading. It represents the total number of tasks here. Maybe "taskCount" is a better name?